packages/synapse-react-client/src/components/ColorGradient/ColorGradient.tsx

```diff
@@ -12,6 +12,8 @@ export function rgba2rgb(background: number[], color: number[]) {
   ]
 }
 
+const VARIANTS_PER_COLOR = 5
+
 export function getColorPalette(
```

Comment:
Made a few modifications to this original function, but its behavior is unchanged